.github/workflows/build-deploy-dev.yml

```diff
@@ -93,6 +93,7 @@ jobs:
       WORKERS_SECRET_NAME: ${{ secrets.OPENSHIFT_SIMS_CREDS_NAME }}
       QUEUE_CONSUMERS_SECRET_NAME: ${{ secrets.OPENSHIFT_SIMS_CREDS_NAME }}
       QUEUE_PREFIX: ${{ secrets.QUEUE_PREFIX }}
+      IER_CRON: ${{ secrets.IER_CRON }}
```

Comment:
I guess that we would have many of those. Maybe we can start with some prefixes like `SCHEDULRS_IER_CRON`.